### PR TITLE
Referenced to the release artifacts of 2.1.0-beta1

### PR DESCRIPTION
**Pull request by vadymzagorodko**
Referenced to the release artifacts of 2.1.0-beta1.

### CODE REVIEW

File: aws/windows-agent/main.tf [hunk 1]
```diff
@@ -34,7 +34,7 @@ module "dcos" {
   ansible_bundled_container = "mesosphere/dcos-ansible-bundle:windows-beta-support"
 
   #Linux Installer path - place url with "pull/PR#" or "master" suffix here:
-  custom_dcos_download_path = "https://downloads.mesosphere.com/dcos-enterprise/testing/master/dcos_generate_config.ee.sh"
+  custom_dcos_download_path = "https://downloads.mesosphere.io/dcos-enterprise/testing/2.1.0-beta1/dcos_generate_config.ee.sh"
```
Comment (fatz) [inline aws/windows-agent/main.tf]: its downloads.mesosphere.com

File: aws/windows-agent/main.tf [hunk 2]
```diff
@@ -47,7 +47,7 @@ enable_windows_agents: true
   #Windows Installer path - place url with "pull/PR#" or "master" suffix here:
 ansible_additional_config = <<-EOF
 dcos:
- download_win: "https://downloads.mesosphere.com/dcos-enterprise/testing/master/windows/dcos_generate_config_win.ee.sh"
+ download_win: "https://downloads.mesosphere.io/dcos-enterprise/testing/2.1.0-beta1/windows/dcos_generate_config_win.ee.sh"
```
Comment (fatz) [inline aws/windows-agent/main.tf]: its downloads.mesosphere.com

File: azure/windows-agent/main.tf [hunk 3]
```diff
@@ -48,7 +48,7 @@ module "dcos" {
   ansible_bundled_container = "mesosphere/dcos-ansible-bundle:windows-beta-support"
 
   #Linux Installer path - place url with "pull/PR#" or "master" suffix here:
-  custom_dcos_download_path = "https://downloads.mesosphere.com/dcos-enterprise/testing/master/dcos_generate_config.ee.sh"
+  custom_dcos_download_path = "https://downloads.mesosphere.io/dcos-enterprise/testing/2.1.0-beta1/dcos_generate_config.ee.sh"
```
Comment (fatz) [inline azure/windows-agent/main.tf]: its downloads.mesosphere.com

File: azure/windows-agent/main.tf [hunk 4]
```diff
@@ -57,7 +57,7 @@ enable_windows_agents: true
   #Windows Installer path - place url with "pull/PR#" or "master" suffix here:
   ansible_additional_config = <<-EOF
 dcos:
-  download_win: https://downloads.mesosphere.com/dcos-enterprise/testing/master/windows/dcos_generate_config_win.ee.sh
+  download_win: "https://downloads.mesosphere.io/dcos-enterprise/testing/2.1.0-beta1/windows/dcos_generate_config_win.ee.sh"
```
Comment (fatz) [inline azure/windows-agent/main.tf]: its downloads.mesosphere.com

**Review by fatz — CHANGES_REQUESTED** [review]

**Comment by DominikDary** [comment]
we should also set the attribute of dcos_version:
`dcos_version              = "2.1.0-beta1"`

File: aws/windows-agent/main.tf (hunk 2 above)
Comment (vadymzagorodko) [reply aws/windows-agent/main.tf]: Updated

File: azure/windows-agent/main.tf (hunk 4 above)
Comment (vadymzagorodko) [reply azure/windows-agent/main.tf]: Updated

**Comment by fatz** [comment]
@vadymzagorodko see @DominikDary's comment

File: azure/windows-agent/main.tf (hunk 3 above)
Comment (vadymzagorodko) [reply azure/windows-agent/main.tf]: fixed

**Review by DominikDary — APPROVED** [review]

**Review by fatz — APPROVED** [review]

**Pull request merged**